tox.ini

```diff
@@ -0,0 +1,34 @@
+[tox]
+envlist = coverage,lint
+
+[gh-actions]
+python =
+    3.6: coverage,lint
+    3.7: coverage
+    3.8: coverage
```

Comment:
@rmotitsuki we've opened a discussion internally (that you've probably seen) to move on from py3.6 and use py3.9 (and possibly 3.10, but 3.10 I think it's more far away since kytos will need more fixes https://github.com/kytos-ng/kytos/issues/201). However if we set version 3.9 here and on `envlist` (a few lines above) to do cov and lint on version `py3.9` then scrutinizer might break, so it'd require a fix on scrutinizer as well. If we stick with 3.6 for now then we'd have to map this, or potentially already address on this PR let me know what you think.